BlueShift-iOS-SDK/BlueShift.m

```diff
@@ -818,4 +816,14 @@ - (BOOL)isBlueshiftUniversalLinkURL:(NSURL *)url {
     }
     return false;
 }
+
+- (BOOL)isBlueshiftPushNotification:(NSDictionary *)userInfo {
+    if (userInfo) {
+        if ([userInfo valueForKey:kInAppNotificationModalMessageUDIDKey]) {
+            return  YES;
+        }
+    }
+    return  NO;
```

Comment:
can we inline this one? something like this (ignore the syntax :) )

```return userInfo != null && [userInfo valueForKey:kInAppNotificationModalMessageUDIDKey] != null```

Reply:
Fixed